### ISSUE TITLE
Changes to the URL structure should result in a review of whether redirects are needed

### ISSUE THREAD

**Issue by davidascher** [mozilla/webmaker.org#1207]
Particularly important as we make changes to the teaching tools. Let's not break links that exist out there.

/cc @secretrobotron

**Comment by secretrobotron** [comment]
@davidascher do you have any specific examples that we can review?

**Comment by davidascher** [comment]
https://github.com/mozilla/webmaker.org/issues/1208

**Comment by hannahkane** [comment]
A little confused about how to make this ticket actionable. Is this a reminder to trigger a review in the future, or a call to automate something now?

**Comment by secretrobotron** [comment]
Right. @davidascher can this close, or are there specifics we can dig into here?

**Comment by davidascher** [comment]
How 'bout:

 -[ ] Make a testsuite for webmaker.org that loops over all current URLs and ensures that none yield 404s.
 -[ ] Make said testsuite trigger on push via travis integration, so that PRs that cause URLs to break break the build.
